### DISCUSSION TITLE
None of the derivative-fields and extension-tasks (on create...) is triggered

### DISCUSSION THREAD

**Discussion by webwurm** [AntlerVC/firetable#1092] [Support / Q&A] [answered]
As I had a problem already a few weeks ago (see [here](https://github.com/rowyio/rowy/discussions/1055)) - and then suddenly everything worked again. Now it reappeared. And I can't figure out why...

I have a Users-table with some derivative-fields that trigger on change of the "time_created"-field and some tasks that trigger when a new entry is created, like creating welcome-messages and such.

My issue is: Nothing seems to trigger anymore when creating a new document in the Users-collection; no tasks are working and the derivative-fields are not calculated until I press "Evaluate".

Compiling works fine. I added "logging.log" in the tasks - but it seems they are not even triggered. The only error I keep finding in the cloud-logs is this:
![213911829-222f5a66-3e1a-4922-9b75-c7eff8b4d8a0](https://user-images.githubusercontent.com/1053176/214508421-fe587adc-f392-4b99-b737-a6ea242fbbf1.jpg)

This error has sadly a heavy impact on me, since all the scripts that are implemented to set up a new user are not working. 
I'd really happy about some help - thank you very much!

**Comment by BeeBombshell** [comment]
Hey @webwurm, I'm sorry you had to go through such trouble. However, I'm not sure why this might be happening. I think it'd be best if you could set up a [quick 1:1 chat](https://rowy.io/meet) with us so that we can help you resolve this.

**Reply by webwurm** [reply]
Thanks! Today the 1:1-chat didn't happen for some reason - I hope on Monday.

In the meantime I figured out that the everything is triggered if I add a row directly in Rowy in the Users-collection.
It is also triggered if I add a document in the Firebase-console.
It is NOT triggered, if I add it via FlutterFlow and its "Auth/Create Account"-action. Wwhich I don't understand, because it basically creates a user and adds a document in the Users-collection. I really don't know how to resolve this...

**Reply by BeeBombshell** [reply]
Hey @webwurm, has this issue been resolved yet? I am assuming this issue would be related to the Firestore Security Rules. It's probably the case where your FlutterFlow app is not able to access the Firestore database. 
It'd be helpful if you could share your Firebase Security Rules set for your project. (found here: https://console.firebase.google.com/project/[project-name]/firestore/rules)

**Reply by webwurm** [reply]
Hi!
It's still there, but I could narrow it down. Also I can't image that the issue has to do with the rules - for testing purposes I set them basically completely open and it didn't change anything.

Further testing has revealed that the issue comes up when I create a new user with FlutterFlow (which also creates a document in the Users-table); after that I delete this user and the document in the Users-collection - and then create a new user with the same email-address. Somehow then Rowy (or the Cloud Functions) don't trigger all the tasks for creating a new document (a User in my case).

It's a case that probably doesn't happen in "real life" very often - but while testing my new-user-scripts the issue appeared.

**Comment by shamsmosowi** [comment]
Hey @webwurm can you include the full error, it looks like something wrong with the axios library, but not sure where exactly

**Reply by webwurm** [reply]
You are right, in the logs I get the following error:
```
{
  "textPayload": "AxiosError: timeout of 1000ms exceeded\n    at RedirectableRequest.handleRequestTimeout (/layers/google.nodejs.yarn/yarn_modules/node_modules/axios/dist/node/axios.cjs:2781:16)\n    at RedirectableRequest.emit (node:events:513:28)\n    at RedirectableRequest.emit (node:domain:552:15)\n    at Timeout.<anonymous> (/layers/google.nodejs.yarn/yarn_modules/node_modules/follow-redirects/index.js:169:12)\n    at listOnTimeout (node:internal/timers:559:17)\n    at processTimers (node:internal/timers:502:7) {",
  "insertId": "63d162c3000e7e6bf5b08c72",
  "resource": {
    "type": "cloud_function",
    "labels": {
      "function_name": "R-Users",
      "region": "europe-west1",
      "project_id": "..."
    }
  },
  "timestamp": "2023-01-25T17:11:31.949867Z",
  "severity": "ERROR",
  "labels": {
    "instance_id": "00c61b117c504e39f2183d508e59a31ac4c9be28ea928613be3d2712bafdae59ec8a757ad4e1d1e9cd185f3773190109a308db2a780139b1fed2",
    "execution_id": "2k8xm9nyz0mp"
  },
  "logName": "projects/.../logs/cloudfunctions.googleapis.com%2Fcloud-functions",
  "trace": "projects/.../traces/958192a52d076ceddc9f81349d17249d",
  "receiveTimestamp": "2023-01-25T17:11:32.152813644Z"
}
```

**Comment by bofeiw** [comment, marked as answer]
Fixed in https://github.com/rowyio/backend/pull/27
